src/bnb_trading/indicators.py

```
@@ -552,167 +570,170 @@
         except Exception as e:
             logger.error(f"Грешка при генериране на ATR сигнал: {e}")
-            return {'signal': 'HOLD', 'strength': 0.0, 'reason': 'Грешка в ATR сигнал'}
+            return {"signal": "HOLD", "strength": 0.0, "reason": "Грешка в ATR сигнал"}
 
     def get_all_indicators_signals(self, df: pd.DataFrame) -> Dict[str, any]:
         """
         Връща сигналите от всички индикатори
-        
+
         Args:
             df: DataFrame с изчислени индикатори
```

Ruff (0.12.2)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)